test/integration/Main.hs

```diff
@@ -66,6 +50,7 @@ main = do
         link cluster
         let baseURL = "http://localhost:1337/"
         manager <- newManager defaultManagerSettings
+        threadDelay 6000000
```

Comment:
why moving the delay here :thinking: ?

Reply:
It didn't work otherwise... The server didn't manage to start before the request was triggered... Moving here fixed it. Perhaps it is the issue with the `Command` ¯\_(ツ)_/¯

Reply:
Hmmmm. No okay. We do need _both_ actually. The extra IO we give to the command executes in a separate thread. So, if you don't have any delay between the moment you launch the command, and the moment you start running the tests, well, the server won't yet be up for the tests.

Reply:
Hm, putting it here:
```haskell
    cardanoWalletLauncher serverPort bridgePort network = Command
        "cardano-wallet-launcher"
        [ "--network", network
        , "--wallet-server-port", serverPort
        , "--http-bridge-port", bridgePort
        ] (threadDelay 6000000)
        Inherit
```
and leaving inside the `startCluster` at the same time, also causes it to fail:
```
       uncaught exception: HttpException
       HttpExceptionRequest Request {
         host                 = "localhost"
         port                 = 1337
         secure               = False
         requestHeaders       = [("Content-Type","application/json"),("Accept","application/json")]
         path                 = "/v2/wallets"
         queryString          = ""
         method               = "POST"
         proxy                = Nothing
         rawBody              = False
         redirectCount        = 10
         responseTimeout      = ResponseTimeoutDefault
         requestVersion       = HTTP/1.1
       }
        (ConnectionFailure Network.Socket.connect: <socket: 11>: does not exist (Connection refused))

```